sdk/src/types/block/output/native_token.rs

```diff
@@ -251,6 +261,12 @@ impl NativeTokens {
     }
 }
 
+impl WorkScore for NativeTokens {
```

Comment:
You can remove this, it's never used in the context of a block

Reply:
Done